### ISSUE TITLE
force version returns "dev"

### ISSUE THREAD

**Issue by diligent176** [ForceCLI/force#458]
I just updated from v0.22.74 to latest v0.22.81, and now **force version** isn't working properly.
It returns the string **dev** instead of the expected **v0.22.81**.

(This seems like the kind of regression that should be caught by basic unit-tests).
![image](https://user-images.githubusercontent.com/7737939/31965964-ea9d200c-b8bd-11e7-9d41-4f2828ea10d6.png)
![image](https://user-images.githubusercontent.com/7737939/31965970-eee985ce-b8bd-11e7-9ac8-886bc0d80ee5.png)

**Comment by dcarroll** [comment]
Yeah, there must be a change in the .yaml file as that is when the version
is set. Thanks for posting, I'll try to take a look Thursday.

**Comment by sfllaw** [comment]
Ping? This is still a problem with v0.22.84.

**Comment by cwarden** [comment]
I broke this in #437, where the `lib` package was introduced.  I've opened #483 to fix it.

Note that travis builds are currently broken.  I'd like to get @dcarroll to sign off on #481, which fixes them (and removes use of his personal S3 buckets for build artifacts).

We're also trying to get ownership of https://force-cli.heroku.com/ transferred.

After these three issues are resolved, we should be able to release new builds with the correct version string.
